pcdsdevices/lens.py

```diff
@@ -286,31 +341,294 @@ def _make_safe(self):
             filt.insert()
             time.sleep(0.01)
         if filt.inserted:
-            print("REMINDER: Beam stop attenuator moved in!")
+            logger.info('Beam stop attenuator moved in!')
             safe = True
         else:
-            print("WARNING: Beam stop attenuator did not move in!")
+            logger.warning('Beam stop attenuator did not move in!')
             safe = False
         return safe
 
 
 class LensStack(LensStackBase):
+    """
+    Class for Be lens.
+
+    Parameters
+    ----------
+    x_prefix : str
+        The EPICS prefix that identifies the x motor.
+    y_prefix : str
+        The EPICS prefix that identifies the y motor.
+    z_prefix : str
+        The EPICS prefix that identifies the z motor.
+    lens_set : list, optional
+        List of lens sets. e.g. [numer1, lensthick1, number2, lensthick2...]
+    z_offset : number
+        Distance from sample to lens_z=0 in meters.
+    z_dir : number
+        1 or -1, represents beam direction wrt z direction.
+    E: number, optional
+        Beam energy
+    att_obj : attenuator object, optional
+    lcls_obj
+        Object that gets PVs from lcls (for energy)
+    mono_obj
+        Object that gets energy from monochromator
+    beamsize_unfocused : float, optional
+        Radial size of x-ray beam before focusing.
+    path : str
+        Path to the file that defines which lenses are being used.
+
+    Examples
+    --------
```

Comment:
Ok so here is where I think I went a bit too far and provided too much information in the Examples docstring... please let me know

Reply:
I think it's good to include verbose docs in a docstring :+1:
The alternative is to move it into the docs/source in a tutorial section or something like that

Reply:
Agreed! These should render just fine in the sphinx-generated docs.